src/core/components/InstallButton/index.js

```diff
@@ -73,6 +91,9 @@ export class InstallButtonBase extends React.Component {
         'InstallButton-button--disabled': buttonIsDisabled,
       }
     );
+    const installURL = findInstallURL({
```

Comment:
I'm glad you are optimistic about the current beta install URL because I am a bit worried tbh. I haven't looked into what's needed though. Ping me if you want to chat about implementation ideas ahead of code review.